tests/test_h5md.py

```diff
@@ -0,0 +1,190 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import numpy as np
+
+from nomad.datamodel import EntryArchive
+from atomisticparsers.h5md import H5MDParser
+
+
+def approx(value, abs=0, rel=1e-6):
+    return pytest.approx(value, abs=abs, rel=rel)
+
+
+@pytest.fixture(scope='module')
+def parser():
+    return H5MDParser()
+
+
+def test_md(parser):
```

Comment:
does this test also traj sampling and stuff? Maybe good to also have test where some quantities are missing like system_info. Not sure if you can generate such a file.

Reply:
yeah, I agree. I would say that I need to generate a whole variety of tests to try to cover as many scenarios as possible, since the file format has so much flexibility. I start working on this..